static/src/javascripts/projects/commercial/modules/comscore.js

```diff
@@ -19,22 +35,33 @@ const getGlobals = (keywords: string): comscoreGlobals => {
     return globals;
 };
 
-export const init = (): Promise<void> => {
-    if (commercialFeatures.comscore) {
+const initOnConsent = (state: boolean | null) => {
+    if (!initialised) {
         // eslint-disable-next-line no-underscore-dangle
         window._comscore = window._comscore || [];
 
         // eslint-disable-next-line no-underscore-dangle
-        window._comscore.push(getGlobals(config.get('page.keywords', '')));
+        window._comscore.push(
+            getGlobals(state, config.get('page.keywords', ''))
```

Comment:
You could pass `state`'s associated boolean value to `getGlobals` by passing `!!state`  instead of `state`, so `null` will become `false`. This will simplify the signature of `getGlobals` and mean you can also remove this `// flowlint-next-line sketchy-null-bool:off` within that function.

Reply:
👍 
I had tried using `!!state` within the function to avoid suppressing the sketchy null error but the prettifier was removing it on save. However I just tried doing it here instead and it doesn't remove the `!!`. I wonder why that is?